pkg/operator/internal/webhooks/kubecarrier_webhook.go

```diff
@@ -70,5 +71,40 @@ func (r *KubeCarrierWebhookHandler) validateCreate(kubeCarrier *operatorv1alpha1
 	if kubeCarrier.Name != constants.KubeCarrierDefaultName {
 		return fmt.Errorf("KubeCarrier object name should be 'kubecarrier', found: %s", kubeCarrier.Name)
 	}
+	auth := map[string]bool{}
```

Comment:
I think in go, we use map[string]struct{} as set in common cases

Reply:
it looks much nicer with `map[string]bool{}` then `map[string]strunct{}{}`.
also with `bool` as a value we can do `if auth['anonymous']` but with `struct` you will have to do `if _, ok := auth['anonymous']; ok{`